sonar-batch/src/main/java/org/sonar/batch/phases/PhaseExecutor.java

```diff
@@ -120,6 +124,13 @@ private void localIssueTracking() {
     localIssueTracking.execute();
     eventBus.fireEvent(new BatchStepEvent(stepName, false));
   }
+  
+  private void issuesCallback() {
```

Comment:
Should only be executed in preview mode. In "publish" mode there is no issue tracking so there are many missing information on issues.